rl_algorithms/common/helper_functions.py

```diff
@@ -88,3 +88,67 @@ def get_n_step_info(
         next_state, done = (n_s, d) if d else (next_state, done)
 
     return reward, next_state, done
+
+
+def infer_leading_dims(tensor, dim):
```

Comment:
I think these methods don't use for common. Should make `recurrent/utils.py` and move them.